runtime/fundamentals/modules.md

```diff
@@ -329,11 +329,39 @@ Limitations:
 
 ### Overriding NPM packages
 
-We plan to support NPM packages with the patch functionality described above,
-but until then if you have a `node_modules` directory, `npm link` can be used
-without change to accheive the same effect. This is typically done with
-`{ "nodeModulesDir": "manual" }` set in the `deno.json` file. See also the
-documentation on [`node_modules`](/runtime/fundamentals/node/#node_modules)
+Deno supports patching npm packages with local versions, similar to how JSR
+packages can be patched. This allows you to use a local copy of an npm package
+during development without publishing it.
+
+To use a local npm package, configure the `patch` field in your `deno.json`:
+
+```json
+{
+  "patch": [
+    "../path/to/local_npm_package"
+  ],
+  // required until Deno 2.3, but it will still be considered unstable
```

Comment:
I believe this is still required as of Deno 2.3. @dsherret do we have plans to stabilize it in this release?

Reply:
It's still required. Didn't have time to stabilize it.